### ISSUE TITLE
LLM tasks are failing and there are no logs or any information

### ISSUE THREAD

**Issue by xyhhx** [nextcloud/llm#55]
I'm trying to use the AI assistant for various tasks:

- From the UI, I'm trying to use _Context Chat_ to ask simple questions
- From the UI, I'm trying to use _Summarize_ to summarize a call's transcript
- From a terminal, I'm trying to use `occ llm:summarize` to summarize a call's transcript

I've tried with both local LLM and OpenAI integration, and it fails in all cases. With the UI, I instantly get a toast-like notification on top-right saying "Assistant error" (nothing in actual notifications):

![image](https://github.com/nextcloud/llm/assets/105510262/d6c7cad8-fbee-40ba-a1e0-bf3b0d783672)

From the terminal, I get the following:

```
LLM process failed: process exited with code 1
```

In all cases, there is nothing in _Administration Settings > Logging_, and running `docker logs nextclouid-aio-nextcloud` just gives me 

```
Task failed 5
Task failed 7
```

How am I supposed to troubleshoot this?

**Comment by marcelklehr** [comment]
hi!
what versions of nextcloud assistant, llm, etc?

**Comment by xyhhx** [comment]
I'm using:

Nextcloud 28.0.3
Assistant 1.0.6
LLM 1.2.1

**Comment by wolfallein** [comment]
Hi,
Could you let me know if you managed to troubleshoot this?
I get the same problem.

Using Docker container.

Nextcloud Hub 7 (28.0.3)
Assistant 1.0.7
Local Large Language Model 1.2.1

![image](https://github.com/nextcloud/llm/assets/15076592/4264b1e1-56fd-4693-9bf1-359e8afb3d24)

**Comment by xyhhx** [comment]
@wolfallein no, I haven't. My settings are the same as yours. Still, there are no errors or relevant logs. Without any of this it's a nightmare to try to debug...

**Comment by wolfallein** [comment]
Hi,
I found some helpful information here: [https://help.nextcloud.com/t/ai-assistant-failing-resolved/172705](https://help.nextcloud.com/t/ai-assistant-failing-resolved/172705)

I ran: `occ maintenance:repair`

and got: 

```
 - Install dependencies for llm app
     - ERROR: Failed to install python dependencies: Failed to create python venv: python3 -m venv ./python returned The virtual environment was not created successfully because ensurepip is not available.  On Debian/Ubuntu systems, you need to install the python3-venv package using the following command.                                                                                                                                                    
      
     apt install python3.11-venv

You may need to use sudo with that command.  After installing the python3-venv package, recreate your virtual environment.                                                                                                     
Failing command: /var/www/html/custom_apps/llm/python/bin/python3  
```

So, I installed the missing package inside the Docker container for testing. After that, I ran `occ maintenance:repair` again, which took some time.

Unfortunately, it still didn't fix the problem on my machine, but it is a step forward and maybe works for you.

**Comment by xyhhx** [comment]
Thanks for that @wolfallein - I found that thread earlier and have tried it already to no avail :face_exhaling: 

In any case, I think the meaning of this ticket is more specifically that no logs or errors are produced rather than trying to get help fixing why my LLM isn't working.
